docs/skillmap/beginner-skillmap.md

```diff
@@ -11,10 +11,57 @@
 
 ### story-activity1
 
-* name: Create a greeting card
+* name: Greeting card
 * type: tutorial
 * description: Make a greeting card and learn Arcade!
 * tags: easy, story, card, creative
+* next: story-activity2
+* next: clicker-activity1
 
 * url: https://github.com/microsoft/pxt-arcade/docs/skillmap/story/card 
-* imageUrl: https://microsoft.github.io/pxt-arcade/docs/static/skillmaps/story/activity1.png
+* imageUrl: https://microsoft.github.io/pxt-arcade/docs/static/skillmaps/story/story-activity1.png
```

Comment:
since we're in pxt-arcade now, you can use relative urls: `imageUrl: /static/skillmaps/story/story-activity1.png` ; the github.io links only work if the repository has them turned out, which pxt-skillmap-sample does but pxt-arcade doesn't